task_4.py

```diff
@@ -0,0 +1,26 @@
+"""
+Написать программу по переводу целого числа из десятичной системы счисления в двоичную.
+Ввод: значение типа <int>
+Вывод: значение типа <int>
+Примеры:
+45
+101101
+3
+11
+2
+10
+
+autopep8 ON
+"""
+n = int(input('Введите число: '))
+
+
+def conv_dec_to_bin(n):
+    bin_num = ''
+    while n > 1:
+        bin_num += str(n % 2)
```

Comment:
Есть ли сделать так:
```suggestion
        bin_num = str(n % 2) + bin_num 
```
то строку не придется разворачивать при возврате их функции